FND/Agreements/Contracts.rdf

```diff
@@ -356,6 +386,38 @@
 		<cmns-av:synonym>master contract</cmns-av:synonym>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-agr-ctr;MilestoneEvent">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-oc;Occurrence"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&cmns-cls;isClassifiedBy"/>
+				<owl:onClass rdf:resource="&fibo-fnd-agr-ctr;ContractMilestone"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;exemplifies"/>
+				<owl:onClass rdf:resource="&fibo-fnd-agr-ctr;ContractMilestone"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>milestone event</rdfs:label>
+		<skos:definition>event related to the status or level of completion of a designated activity, task, or segment of work required for contract fulfillment</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-agr-ctr;MilestoneSchedule">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-fd;Schedule"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&cmns-col;hasConstituent"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-agr-ctr;MilestoneEvent"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>milestone schedule</rdfs:label>
+		<skos:definition>schedule of milestone events, observations, or other occurrences and the associated dates and/or times when they will be done</skos:definition>
```

Comment:
It may be either an ad hoc schedule or regular schedule, but probably an ad hoc schedule. But there will be other events (payments, rate changes, collateral requirement changes, etc. that will have dates relative to these milestone dates, and this will enable lining the schedules up.